### ISSUE TITLE
Accessibility code should use AccessibleNode

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#132]
The code related to accessibility should use the new AccessibleNode that was recently checked into scenery.  This will help us test out this new type, but also remove a large amount of boiler plate code that was previously required for setting accessible content.

This should be done before #128.

**Comment by jessegreenberg** [comment]
Done, through the above commits - here is a diff of the HTML content from before and after -
![capture](https://cloud.githubusercontent.com/assets/6396244/21662083/20a27aea-d295-11e6-965a-ed66603871a9.PNG)

As far as I can tell, the only difference is that the range inputs and their labels are wrapped in divs now (other than from a few id differences on the html elements)

**Comment by jessegreenberg** [comment]
Ah, the form is also missing aria-describedby.

**Comment by jessegreenberg** [comment]
Alright, all done, except for a couple of common code components that have custom accessible content.  The accessible content used here should probably replace the accessible content that is currently being used in common code, but I would like to sort out https://github.com/phetsims/scenery/issues/596 before that is done.  Closing this issue.
